NPM-Behind-Proxy.md

```diff
@@ -0,0 +1,40 @@
+# NPM Behind A Proxy Server
+
+### Use-cases
+
+You might need to modify `npm install` commands that accesses remote repositories ([npmjs](https://www.npmjs.com/), for example) for installing Node JS modules; if your internet access is through a [proxy server](https://en.wikipedia.org/wiki/Proxy_server).
+
+It's quite common if you are a student in college, or maybe you are a working professional; accessing the internet behind an institute/company mandated firewall.
```

Comment:
Maybe reword to something like: 'Proxy servers are common in college and business type environments.'